twpca/model.py

```diff
@@ -55,26 +86,154 @@ def __init__(self, n_components, shared_length=None,
         }
 
```

Comment:
put code that creates tensorflow variables in a `with` block using `self._graph` as default